packages/flutter/test/foundation/diagnostics_json_test.dart

```diff
@@ -234,7 +232,7 @@ class _TestElement extends Element {
 
 class TestTree extends Object with DiagnosticableTreeMixin {
   TestTree({
-    this.name,
+    this.name = '',
     this.style,
```

Comment:
Why can't this be initialized with `DiagnosticsTreeStyle.none`? Then it could be non-nullable.

Reply:
I don't think it should. According to its `debugFillProperties`, `TestTree.style` is an optional value to override `properties.style`, with `null` standing for "do not override".